Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM bryanlatten/dcos-jenkins-dind-agent:0.5.0-alpine
+FROM bryanlatten/dcos-jenkins-dind-agent:0.6.0-alpine
 
 RUN apk add --upgrade py-pip make
 RUN pip install --upgrade awscli docker-compose
+
+RUN apk update curl jq
```

Comment:
any reason to not include this in the above statements (which can be done in a single step, too)?

Reply:
image caching locally. I really didn't want to wait for pip & w/e else that relied on it to install

Reply:
that's a one-time cost, right? could we optimize for runtime